.github/workflows/github_release.yaml

```diff
@@ -0,0 +1,133 @@
+        # Archive artifact
+      - name: 'Tar router files'
+        run: tar -czvf prime-router-build.tar.gz build
+      - name: Upload Router Artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: prime-router-build-${{ needs.wf_vars.outputs.version }}
+          path: prime-router/prime-router-build.tar.gz
+          retention-days: 1
+
+  build_frontend_react_gh_release:
+    name: "GitHub Release: Build Frontend"
+    needs:
+      - wf_vars
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: frontend-react
+    steps:
+      - name: "Check out changes"
+        uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }} yarn
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - run: yarn install
+      - name: Build Frontend
+        run: yarn run build
+        # Archive artifact
+      - name: 'Tar frontend files'
+        run: tar -czvf static-website-react.tar.gz build
+      - name: Upload Frontend Artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: static-website-react-${{ needs.wf_vars.outputs.version }}
+          path: frontend-react/static-website-react.tar.gz
+          retention-days: 1
+
+  publish_github_release:
+    name: "Publish to Github"
+    needs:
+      - wf_vars
+      - build_router_gh_release
+      - build_frontend_react_gh_release
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Check out changes"
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Download Router Artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: prime-router-build-${{ needs.wf_vars.outputs.version }}
+          path: archives/
+      - name: Download Frontend Artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: static-website-react-${{ needs.wf_vars.outputs.version }}
+          path: archives/
+      - name: 'Check if tags exist'
+        id: tagged
+        shell: bash
+        run: |
+          count=$(git tag --list 'v-*' | wc -l)
+          if [ $count -gt 0 ]; then 
+            echo "::set-output name=tagged::1"; 
+          else 
+            echo "::set-output name=tagged::0"; 
+          fi
+      - name: Build changelog if tags exist
+        if: steps.tagged.outputs.tagged == 1
+        uses: mikepenz/release-changelog-builder-action@v2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Create release
+        uses: ncipollo/release-action@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          tag: ${{ needs.wf_vars.outputs.version }}
+          artifacts: "archives/*.tar.gz,LICENSE"
+          generateReleaseNotes: true
+      - name: Delete Older Releases
+        uses: dev-drprasad/delete-older-releases@v0.2.0
+        with:
+          keep_latest: 5
+          delete_tags: true
```

Comment:
Does this delete the tags from Github? If so, I'm not sure we want to do that, particularly if we're going to be correlating those tags to a changelog.